ReactAndroid/src/main/java/com/facebook/react/modules/timepicker/TimePickerDialogFragment.java

```diff
@@ -52,12 +60,43 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
           DateFormat.is24HourFormat(activityContext));
     }
 
-    return new DismissableTimePickerDialog(
-        activityContext,
-        onTimeSetListener,
-        hour,
-        minute,
-        is24hour);
+    TimePickerDialog dialog = new DismissableTimePickerDialog(
```

Comment:
Maybe do this in the last else block to avoid unnecessarily instantiating it?

Reply:
True! I change this :)